_maps/map_files/Theseus/TGS_Theseus.dmm

```diff
@@ -15786,6 +15762,15 @@
 	},
 /turf/open/floor/plating/almayer,
 /area/almayer/engineering/upper_engineering)
+"aGT" = (
+/obj/effect/attach_point/electronics/dropship1{
+	dir = 1
+	},
+/turf/open/floor/plating,
+/turf/closed/shuttle/dropship1/transparent{
+	icon_state = "90"
+	},
+/area/shuttle/drop1/sulaco)
```

Comment:
Are these changes intentional?

Reply:
Seems like so, I mapped in one cryopod console for each squad in their prep rooms.